Corzbank.Web/ClientApp/src/app/header/wallet/wallet.component.ts

```diff
@@ -57,13 +82,34 @@ export class WalletComponent implements OnInit {
 		return this.transferForm.get("amount");
 	}
 
+	get cardType() {
+		return this.createCardForm.get("cardType");
+	}
+
+	get paymentSystem() {
+		return this.createCardForm.get("paymentSystem");
+	}
+
+	get secretWord() {
+		return this.createCardForm.get("secretWord");
+	}
+
 	ngOnInit(): void {
 		this.cardService.cardSubject.subscribe((data: any) => {
 			this.cards = data;
 		});
 
+		this.currentUserId = JSON.parse(
+			this.storageService.getItem(
+				StorageTypeEnum.LocalStorage,
+				Constants.UserIdKey
+			)
+		);
```

Comment:
How is it possible to have null in a number type? You should validate that during receiving for local storage

Reply:
resolved